### PR TITLE
Arcsight Logger - Improved documentation of the local_search argument

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32711

## Description
Improved documentation of the local_search argument in both relevant commands.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ArcSightLogger/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ArcSight Logger
+- ￿Improved the documentation of the *local_search* argument in the ***as-search-events*** command.
```
Comment (teizenman) [inline Packs/ArcSightLogger/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Improved the documentation of the *local_search* argument in the ***as-search-events*** command.
```

File: Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml [hunk 2]
```diff
@@ -647,7 +647,7 @@ script:
       predefined:
       - "true"
       - "false"
-      description: Indicates the search is local only, and does not include peers. The default value for this parameter is 'true'.
+      description: Indicates the search is local only, and does not include peers. The default value for this parameter is 'true'. If the local search is configured to 'False' in your instance, make sure to add this argument with the 'false' value to the command.
```
Comment (teizenman) [inline Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml:650]: I think you need to clarify which instace you are referring here.
Youre talking about the ArcSight Logger right? not the XSOAR instance of it.

File: Packs/ArcSightLogger/ReleaseNotes/1_0_1.md (hunk 1 above)
Comment (amshamah419) [inline Packs/ArcSightLogger/ReleaseNotes/1_0_1.md]:
```suggestion
- Improved the documentation for the *local_search* argument of the ***as-search-events*** command.
```

File: Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml (hunk 2 above)
Comment (amshamah419) [inline Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml]:
```suggestion
      description: Indicates the search is local only and does not include peers. The default value for this parameter is 'true'. If the local search option is configured to 'False' on your ArcSight Logger instance, make sure to include this argument with a value of 'false' for the command.
```

File: Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml [hunk 3]
```diff
@@ -773,7 +773,7 @@ script:
...
```
Comment (amshamah419) [inline Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml]:
```suggestion
      description: Indicates the search is local only and does not include peers. The default value for this parameter is 'true'. If the local search option is configured to 'False' on your ArcSight Logger instance, make sure to include this argument with a value of 'false' for the command.
```

**Comment by amshamah419** [comment]
@reutshal See my suggestions

File: Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml [hunk 4]
```diff
@@ -259,7 +259,7 @@ script:
             bodyArgs.summary_fields = parseArray(summaryFields);
         }
         if (fieldSummary) {
-            bodyArgs.field_summary = fieldSummary;
+            bodyArgs.field_summary = parseBool(fieldSummary);
```
Comment (teizenman) [inline Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml:262]: Add this to the RN as well

**Comment by reutshal** [comment]
@amshamah419 Mind to take another look at the RN? I've added a point

**Review by teizenman — APPROVED** [review]

**Comment by amshamah419** [comment]
> @amshamah419 Mind to take another look at the RN? I've added a point

Changes look good

**Pull request merged**